dist/css/uswds.css

```diff
@@ -1381,6 +1383,8 @@ textarea {
 }
 
 select {
+  padding-right: 32px;
```

Comment:
Can you change this value to `3.2rem`? We only use px for detail work when it's 5px or less.